main/src/ca/uwaterloo/flix/language/ast/WeededAst.scala

```diff
@@ -247,6 +247,19 @@ object WeededAst {
 
   }
 
+  sealed trait ExtPattern {
```

Comment:
Since match stuff is before ext stuff it would make sense that normal patterns are before ExtPattern

Reply:
(I suppose this comment applies to all ASTs)